src/Common/src/Interop/Windows/mincore/Interop.GetConsoleTitle.cs

```diff
@@ -20,13 +20,22 @@ internal partial class mincore
         // throw.
         private const int MaxAllowedBufferSizeInChars = 24500;
 
+        // GetConsoleTitle sometimes interprets the second parameter (nSize) as number of bytes and sometimes as the number of chars.
+        // Instead of doing complicated and dangerous logic to determine if this may or may not occur,
+        // we simply assume the worst and reserve a bigger buffer. This way we may use a bit more memory,
+        // but we will always be safe.
+        private static int AdjustNumberOfChars(int numChars)
```

Comment:
Should this be called CharCountToByteCount, or something like that?  It's not clear otherwise what kind of "adjusting" is being done.